core/flow/rule_manager.go

```diff
@@ -207,24 +294,40 @@ func IsValidFlowRule(rule *FlowRule) error {
 		return err
 	}
 
-	return checkControlBehaviorField(rule)
+	if err := checkStatisticConfig(rule); err != nil {
+		return err
+	}
+
+	return checkControlStrategyField(rule)
 }
 
-func checkClusterField(rule *FlowRule) error {
+func checkClusterField(rule *Rule) error {
```

Comment:
I think we'd better judge the two parameters separately for more cluster parameters in the future

Reply:
Maybe we could improve here in future practical scenarios.
Currently, I think it's ok.